sycl/source/detail/device_impl.cpp

```diff
@@ -239,6 +239,8 @@ bool device_impl::has(aspect Aspect) const {
     return has_extension("cl_khr_int64_base_atomics");
   case aspect::int64_extended_atomics:
     return has_extension("cl_khr_int64_extended_atomics");
+  case aspect::atomic64:
+    return get_info<info::device::atomic64>();
```

Comment:
what is the definition of this "atomic64" feature? please add doc. does it not map to any existing features, e.g. "int64_extended_atomics" just above?

Reply:
Please see my comment above.  As I understand it aspect::int64_extended_atomics is not a SYCL2020 feature.

Reply:
Doc: https://www.khronos.org/registry/SYCL/specs/sycl-2020/html/sycl-2020.html#_sycl_support_of_non_core_opencl_features